apps/ocean-api/__tests__/modules/PlaygroundModule.test.ts

```diff
@@ -0,0 +1,48 @@
+import { OceanApiTesting } from '../../testing/OceanApiTesting'
+import waitForExpect from 'wait-for-expect'
+
+describe('regtest', () => {
+  const apiTesting = OceanApiTesting.create()
+
+  beforeEach(async () => {
+    apiTesting.playgroundEnable(false)
+    await apiTesting.start()
+  })
+
+  afterEach(async () => {
+    await apiTesting.stop()
+  })
+
+  it('block count should not increment', async () => {
+    const initial = await apiTesting.rpc.blockchain.getBlockCount()
+
+    await new Promise((resolve) => {
+      setTimeout(_ => resolve(0), 5000)
+    })
```

Comment:
`question:` Could a test utility box with a method be valid here for similar use cases in the future?
```ts
async function waitForTimeout(ms: number) {
  await new Promise((resolve) => {
    setTimeout(_ => resolve(0), ms)
  })
}
```
I am still working on catching up on project conventions so posing this as a question for further elaboration and knowledge gains.

Reply:
Hmm, I am a little skeptical doing this now since it's one such pattern. But definetly something to revisit later.

Reply:
Preferably (my pref) to stick with more native syntax in the test itself. Especially for simple stuff, to limit the "black box" from growing.

Reply:
Thanks for the feedback, will mentally log it and move on! 😄